js-inputs-and-strings_switch-values/js/index.js

```diff
@@ -24,3 +26,18 @@ uppercaseButton.addEventListener("click", () => {
 lowercaseButton.addEventListener("click", () => {
   firstInput.value = firstInput.value.toLowerCase();
 });
+
+uppercaseButton.addEventListener("click", () => {
+  secondInput.value = secondInput.value.toUpperCase();
+});
+
+lowercaseButton.addEventListener("click", () => {
+  secondInput.value = secondInput.value.toLowerCase();
+});
+
+switchButton.addEventListener("click", () => {
+  firstInput.value = [
```

Comment:
How does this work? 🤔 

But there's also more simple way to do this

This could be simplified I think

```
const firstInputValue = firstInput.value;
const secondInputValue = secondInput.value;
firstInput.value = secondInputValue;
secondInput.value = firstInputValue;
```

Reply:
true, thats simpler that the array methode.